airbyte-integrations/bases/source-acceptance-test/unit_tests/test_core.py

```diff
@@ -0,0 +1,56 @@
+#
+# MIT License
+#
+# Copyright (c) 2020 Airbyte
+#
+# Permission is hereby granted, free of charge, to any person obtaining a copy
+# of this software and associated documentation files (the "Software"), to deal
+# in the Software without restriction, including without limitation the rights
+# to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+# copies of the Software, and to permit persons to whom the Software is
+# furnished to do so, subject to the following conditions:
+#
+# The above copyright notice and this permission notice shall be included in all
+# copies or substantial portions of the Software.
+#
+# THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+# IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+# FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+# AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+# LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+# OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+# SOFTWARE.
+#
+from unittest.mock import MagicMock
+
+import pytest
+from airbyte_cdk.models import AirbyteMessage, Type
+from source_acceptance_test.tests.test_core import TestDiscovery as _TestDiscovery
+
+
+@pytest.mark.parametrize(
+    "schema, cursors, is_good",
+    [
+        ({}, ["created"], False),
+        ({"properties": {"created": {"type": "string"}}}, ["created"], True),
+        ({"properties": {"created_at": {"type": "string"}}}, ["created"], False),
+        ({"properties": {"created": {"type": "string"}}}, ["updated", "created"], False),
+        ({"properties": {"created": {"type": "string"}, "updated": {"type": "string"}}}, ["updated", "created"], True),
```

Comment:
Again, default_cursor_field may represent the cursor path, not 2 different values.

Reply:
Fixed